src/polycubed/src/rest_server.cpp

```diff
@@ -152,6 +157,59 @@ void RestServer::shutdown() {
   }
 }
 
+void RestServer::create_cubes(json &cubes) {
+  /*
+   * This is possible that a cube's port has as peer another port
+   * of a cube that hasn't been created yet.  This logic removes all
+   * the peers from the request and sets them after all cubes have
+   * been created.
+   */
+  auto peers = utils::strip_port_peers(cubes);
+  std::vector<Response> resp = {{ErrorTag::kNoContent, nullptr}};
+  bool error = false;
+
+  for (auto &cube : cubes) {
+    auto &s = core.get_service_controller(cube["service-name"]);
+    auto res = s.get_management_interface()->get_service()
+            ->CreateReplaceUpdate(cube["name"], cube, false, true);
+
+    if (res.size() != 1 || res[0].error_tag != kCreated) {
+      std::string msg;
+      for (auto & r: res) {
+        //logger->debug("- {}", r.message);
+        msg += std::string(r.message) + " ";
+      }
+
+      throw std::runtime_error("Error creating cube: " + cube["name"].get<std::string>() + msg);
+    }
+  }
+
+  for (auto &[peer1, peer2] : peers) {
+    core.try_to_set_peer(peer1, peer2);
+  }
+}
+
+void RestServer::load_last_topology() {
+  logger->info("loading cubes from {}", configuration::config.getCubesDumpFilePath());
+  std::ifstream topologyFile(configuration::config.getCubesDumpFilePath());
+  if (!topologyFile.is_open()) {
+    logger->warn("error opening dump file: {}", configuration::config.getCubesDumpFilePath());
```

Comment:
I think this message can be deleted, if the file is not present it means there is nothing to be loaded.